src/pages/api/articles/[id].tsx

```diff
@@ -0,0 +1,12 @@
+import { articles } from "../../../../data";
+
+export default ({ query: { id } }, res) => {
+  const filtered = articles.filter((a) => a.id === id);
+
+  // User with id exists
+  if (filtered.length > 0) {
+    res.status(200).json();
+  } else {
```

Comment:
elseは死語なので早期リターンした方がいい
```suggestion
  return;
}
```